src/features/user/Settings/EditProfile/AddressManagment/index.tsx

```diff
@@ -0,0 +1,53 @@
+import type { Address } from '@planet-sdk/common';
+import type { AddressAction } from '../../../../common/types/profile';
+
+import { useMemo, useState } from 'react';
+import { useTranslations } from 'next-intl';
+import AddressList from './microComponents/AddressList';
+import { useUserProps } from '../../../../common/Layout/UserPropsContext';
+import WebappButton from '../../../../common/WebappButton';
+import styles from './AddressManagement.module.scss';
+import CenteredContainer from '../../../../common/Layout/CenteredContainer';
+
+export const addressTypeOrder = ['primary', 'mailing', 'other'];
+const AddressManagement = () => {
+  const { user } = useUserProps();
+  const tProfile = useTranslations('Profile.addressManagement');
+  const [userAddresses, setUserAddresses] = useState<Address[]>(
+    user?.addresses
+  ); // need to update planet-sdk to include addresses key
+  const [addressAction, setAddressAction] = useState<AddressAction | null>(
+    null
+  );
+  const toggleAddAddressModal = () => {};
+  const sortedAddresses = useMemo(() => {
+    return userAddresses.sort((a, b) => {
+      return (
+        addressTypeOrder.indexOf(a.type) - addressTypeOrder.indexOf(b.type)
+      );
+    });
+  }, [userAddresses]);
+
+  return (
+    <section className={styles.addressManagement}>
+      <h2 className={styles.addressManagementTitle}>
```

Comment:
There is no `addressManagement` class defined in the scss file, so nothing appears in the browser inspector.

I'd suggest moving some styles from the `addressManagementTitle`

e.g.
```
.addressManagement {
	display: flex;
	flex-direction: column;
	margin-top: 40px;
	gap: 20px;
}

.addressManagementTitle {
  font-size: $fontLarge;
  font-weight: 600;
  margin: 40px 0 20px; //Remove this
}
```